tests/unit/backend/corpora/api_server/test_v1_dataset.py

```diff
@@ -212,6 +212,34 @@ def test__enrich_tissue_with_ancestors_missing_key_ok(self):
         Dataset.enrich_tissue_with_ancestors(dataset)
         self.assertNotIn("tissue_ancestors", dataset)
 
+    def test__enrich_cell_type_with_ancestors_expands_correctly(self):
+        dataset = {"cell_type": [{"ontology_term_id": "CL:0000738", "label": "Test"}]}
+        Dataset.enrich_cell_type_with_ancestors(dataset)
+        self.assertIn("cell_type_ancestors", dataset)
+        self.assertEqual(
+            dataset["cell_type_ancestors"],
+            [
+                "CL:0000255",
+                "CL:0002371",
+                "CL:0000988",
+                "CL:0000738",
+                "CL:0000548",
+                "CL:0000219",
+                "CL:0000003",
+                "CL:0002242",
+            ],
+        )
+
+    def test__enrich_cell_type_with_ancestors_empty_key_ok(self):
```

Comment:
curious, will we ever have this case? And if so, shouldn't it be an error since no datasets should contain empty cell types